build/__pythonVersions.sh

```diff
@@ -1,11 +1,11 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PYTHON_RUNTIME_BASE_TAG='20230208.7'
+PYTHON_RUNTIME_BASE_TAG='20230209.1'
```

Comment:
This base tag is from an Oryx-BaseImages-Python pipeline invocation that ran with 3.11.1 removed.